### ISSUE TITLE
Test that landmarks are appropriately labeled across admin

### ISSUE THREAD

**Issue by kimisgold** [omeka/Omeka#1103]
The [`landmark-labels`](https://github.com/omeka/Omeka/tree/landmark-labels) branch has corrections for various landmarks, mainly `<nav>` elements. Test that they are properly read by screen readers. I'm also taking feedback for vague or inaccurate labels. Here are the affected landmarks:

- Admin side
  - [x] Appearance nav
  - [x] Content nav
  - [x] Admin nav
  - [x] Top search form
  - [x] Items only search form
  - [x] Items nav (across browse, tags, and advanced item search)
  - [x] Header (no longer has extraneous "banner" role)
- Public side
  - [ ] Pagination nav
  - [ ] Footer nav

**Comment by kimisgold** [comment]
I've updated the branch to have labels for both the installation-wide search and the items only search.

**Comment by aviannamiller** [comment]
Feedback for admin-side landmark edits:

- Items search form looks good, although to avoid repetition, I suggest removing the label on the main content area on this page that references the same heading ("Search items"). Up to you if you want to make the label on the search just read "Items", but I understand it's referencing the page and form as a whole, so I do see either argument.
- I see the nav tabs have been wrapped under a nav landmark for Appearance, but in other instances of the same pattern, this is not the case. `#section-nav` tabs appear in settings, user edit, and resource edit pages, to name just those that are part of the core. For the first two, the proper role is there, but no distinguishing label. The latter has a list role, no label.
- For landmarks containing duplicate content, in this case "top pagination" and "bottom pagination", I would recommend making these labels the same (probably just "pagination"?). See [General Principles of Landmark Design, Step 3](https://www.w3.org/WAI/ARIA/apg/practices/landmark-regions/#generalprinciplesoflandmarkdesign). Also, I suppose we could be updating our labels for pagination controls along the same sort of guidance provided here (could be "Search results", "Items browse", etc.). Same for S.

**Comment by aviannamiller** [comment]
For public-side content, the following comments on the pagination and footer nav, followed by additional feedback on other landmarks.

- Pagination labels could be updated to reflect purpose, see previous comment on this.
- Seasons, Santa Fe, Minimalist: labels distinguishing top and bottom pagination should be changed to the same thing for both locations.
- Berlin, Center Row, Emiglio, Minimalist: I'm not seeing a specific label on the footer nav. Since there can often be a slightly different collection of links here than in the main nav, I would think there should be some sort of label on these, though I'm not set on what that would be (e.g. "Sitemap"... not sure of the accuracy?).
- Freedom or Lively: Within footer menus, no nav elements, and thus no labels

Other notes:
- Often, there are some strange screen reader announcements on the site search. It seems to be something to do with the form having the label of the site title but being separate to the search. I think there are one or two themes (such as Big Picture) where this is not the case, but for the most part, can the search role move on to the form to try to consolidate labelling? Perhaps there's a better approach, but I think this should just generally be revisited.
- Thanks, Roy, Berlin, Seasons, Big Picture, Center Row, Emiglio, Santa Fe, Minimalist, Rhythm, Freedom, Lively: no nav element present for page breadcrumbs
- Main nav
  - Thanks, Roy, Berlin, Seasons: “Main navigation” could lose “navigation”
  - Rhythm could lose duplicate “Navigation” label
  - Center Row, Emiglio, Santa Fe, Minimalist, Freedom, Lively: Currently, no “main” label at all
- “Items navigation” could lose “navigation” for Thanks, Roy and Seasons
  - Berlin, The Daily, Big Picture, Center Row, Foundation, Freedom, Lively: Comparable labels are not present at all within browse header
  - Emiglio: No nav landmark here at all
- Big Picture: “Site header” is unnecessary on banner
- Foundation: Mobile nav could lose extra “navigation” label
- Freedom, Lively: Lack of distinguishing label on mobile nav

**Comment by kimisgold** [comment]
> Items search form looks good, although to avoid repetition, I suggest removing the label on the main content area on this page that references the same heading ("Search items"). Up to you if you want to make the label on the search just read "Items", but I understand it's referencing the page and form as a whole, so I do see either argument.

I've simplified the search's `aria-label` to just "Items".

> For landmarks containing duplicate content, in this case "top pagination" and "bottom pagination", I would recommend making these labels the same (probably just "pagination"?). See [General Principles of Landmark Design, Step 3](https://www.w3.org/WAI/ARIA/apg/practices/landmark-regions/#generalprinciplesoflandmarkdesign). Also, I suppose we could be updating our labels for pagination controls along the same sort of guidance provided here (could be "Search results", "Items browse", etc.). Same for S.

These have all been simplified to "Pagination."

> For landmarks containing duplicate content, in this case "top pagination" and "bottom pagination", I would recommend making these labels the same (probably just "pagination"?). See [General Principles of Landmark Design, Step 3](https://www.w3.org/WAI/ARIA/apg/practices/landmark-regions/#generalprinciplesoflandmarkdesign). Also, I suppose we could be updating our labels for pagination controls along the same sort of guidance provided here (could be "Search results", "Items browse", etc.). Same for S.

For this branch, I'm focusing on some low hanging fruit to make it into the next bugfix release. There's definitely more work to do filling gaps, but I think we can complete it in multiple phases. Would you say this branch is ready to merge? I'm looking for it to be improvements that don't introduce new problems. Updating the themes and addressing the more complicated approach to tabs would likely be their own issue.

**Comment by aviannamiller** [comment]
> For this branch, I'm focusing on some low hanging fruit to make it into the next bugfix release. There's definitely more work to do filling gaps, but I think we can complete it in multiple phases. Would you say this branch is ready to merge? I'm looking for it to be improvements that don't introduce new problems. Updating the themes and addressing the more complicated approach to tabs would likely be their own issue.

Certainly, changes look good. This can definitely be merged at this point and the rest revisited at a later date (as well as in a new issue, if you prefer).
